src/main/java/hudson/plugins/analysis/core/BuildResult.java

```diff
@@ -225,8 +252,8 @@ protected BuildResult(final AbstractBuild<?, ?> build, final BuildHistory histor
      *            the build to start with
      * @return the history
      */
-    protected BuildHistory createHistory(final AbstractBuild<?, ?> build) {
```

Comment:
Strict binary compatibility would retain this method as a chained call through to the new one.